### PR TITLE
{date}[intel/2015b] h5py 2.5.0 w/ HDF5 1.8.15-patch1 and Python 2.7.10 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

File: easybuild/easyconfigs/h/h5py/h5py-2.5.0-intel-2015b-Python-2.7.10-HDF5-1.8.15-patch1.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+name = 'h5py'
+version = '2.5.0'
+
+homepage = 'http://www.h5py.org/'
+description = """HDF5 for Python (h5py) is a general-purpose Python interface to the Hierarchical Data Format library,
+ version 5. HDF5 is a versatile, mature scientific software library designed for the fast, flexible storage of enormous
+ amounts of data."""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+toolchainopts = {'usempi': True}
```
Comment (wpoely86) [inline easybuild/easyconfigs/h/h5py/h5py-2.5.0-intel-2015b-Python-2.7.10-HDF5-1.8.15-patch1.eb]: also add one with mpi. For my personal use cases, mpi is an annoyance here.

Comment (boegel) [inline easybuild/easyconfigs/h/h5py/h5py-2.5.0-intel-2015b-Python-2.7.10-HDF5-1.8.15-patch1.eb]: This is with MPI. Do you mean without MPI?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/98881781a1e0aa863af9 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 2 out of 3 (3 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/110e0fd6bf682835e89f for a full test report.

File: easybuild/easyconfigs/h/h5py/h5py-2.5.0-intel-2015b-Python-2.7.10-HDF5-1.8.15-patch1.eb (hunk 1 above)
Comment (wpoely86) [inline easybuild/easyconfigs/h/h5py/h5py-2.5.0-intel-2015b-Python-2.7.10-HDF5-1.8.15-patch1.eb]: Yes, of course. This is really annoying if you don't use MPI in your code. So please add a non-mpi version.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/12b4eb7eebe94c2187c2 for a full test report.

File: easybuild/easyconfigs/h/h5py/h5py-2.5.0-intel-2015b-Python-2.7.10-HDF5-1.8.15-patch1.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/h/h5py/h5py-2.5.0-intel-2015b-Python-2.7.10-HDF5-1.8.15-patch1.eb]: done

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 4 (4 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.10
See https://gist.github.com/8c672c9eb83936e2f6bf for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/37c91fc4eafa7ad00e90 for a full test report.

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**